Src/GUI/Plugins/PlotterPlugin.py

```diff
@@ -487,7 +487,7 @@ def on_load_data(self, event=None):
                         data[k]['axis'] = []
                 else:
                     data[k]['axis'] = []
-                data[k]['data'] = _vars[k].getValue()
+                data[k]['data'] = _vars[k][:]
                 data[k]['units'] = getattr(_vars[k],"units","au")
             
             unique_name = self.unique(basename, self.plugin._dataDict)
```

Comment:
Also, why is `basename` defined inside the for loop, but referenced only outside of it?

Reply:
No, basename is defined at the `for i in range(len(filelist))` level and use at the end of that block. I do not see any problem here.